src/components/App/App.js

```diff
@@ -2,17 +2,33 @@ import './App.css';
 import Button from '../Button/Button';
 import CardList from '../CardList/CardList';
 import Section from '../Section/Section';
+import Footer from '../Footer/Footer';
+import messages from '../../assets/local/en';
 import { benefits } from '../../utils/config';
 
 function App() {
   return (
     <>
-      <Button label={'Sample Button'}/>
-      <Section type='benefits' layout='four-columns' title='Benefits'>
-        <CardList cards={benefits}></CardList>
+      <link
```

Comment:
I wonder, could this `<link>` tag and the following `<script>` tag be moved to `<head>` of  `/public/index.html`?

Reply:
I just forgot to transfer <link> and <script>